### DISCUSSION TITLE
Jaeger and ScyllaDB Integration

### DISCUSSION THREAD

**Discussion by guy9** [jaegertracing/jaeger#4679] [Show and tell]
I co-wrote a blog post about [using Jaeger with ScyllaDB as the storage backend](https://scylladb.medium.com/jaeger-and-scylladb-integration-high-performance-at-scale-2134087eb522) to achieve high performance. 
Any input is appreciated, thanks!

**Comment by yurishkuro** [comment]
Thanks for sharing!
